plugins/events/create.js

```diff
@@ -79,6 +79,11 @@ module.exports = () => ({
 
             if (creator) {
                 payload.creator = creator;
+            } else if (scope.includes('pipeline')) {
```

Comment:
Creator setting is in payload, then it gets preference over "Pipeline Access Token".